crates/robonode-client/src/authenticate.rs

```diff
@@ -51,19 +54,32 @@ pub enum AuthenticateError {
     /// The provided liveness data was invalid.
     #[error("invalid liveness data")]
     InvalidLivenessData,
+    /// The person was not found, it is likely because they haven't enrolled first, no blob.
+    #[error("person not found, no blob")]
+    PersonNotFoundNoBlob,
```

Comment:
```suggestion
    /// The person was not found, it is likely because they haven't enrolled first, no blob.
    #[error("person not found")]
    PersonNotFoundNoBlob,
```

We can call those variants `NoBlob`, but we should avoid adding a different error message.

Also, to avoid confusion that causes this bikeshedding in the first place, I suggest that we add though comments on what is the reason for not having a blob here.

We *must* also somehow indicate that the `NoBlob` version is coming from the older - and soon-to-be phased out - design.

Reply:
I do, but PR is still in draft and so far to be completed, FYI I'm identifying the "kind" of error by message and also checking error data.

Reply:
This was already pre-discussed with @sesh92